README.md

```diff
@@ -2,6 +2,16 @@
 
 Drop it allows one click inserts/uploads of photos among the thousands of photos available on <a href="https://unsplash.com">unsplash.com</a> without leaving the Gutenberg editor.
 
+## How to
+
+After installing please run the following commands to make it work : 
+
+`npm install && npm run build`
+
+Please follow this tutorial https://youknowriad.github.io/gcf/#/contribute if you want to contribute.
```

Comment:
I think we should remove this line. The sentence above should be enough for now. If the plugin becomes bigger, we'll have to add docs in a similarily to GCF.

Reply:
makes sense